website/addons/box/serializer.py

```diff
@@ -0,0 +1,110 @@
+from website.addons.base.serializer import OAuthAddonSerializer
+
+from website.util import api_url_for, web_url_for
+
+from box.client import BoxClient, BoxClientException
+
+
+class BoxSerializer(OAuthAddonSerializer):
+
+    def credentials_owner(self, user_settings=None):
+        return user_settings.owner or self.user_settings.owner
+
+    def serialize_folder(self, folder):
```

Comment:
@mfraezz can you confirm whether or not this function is being used? I only see it used in it's test, so I think it can be removed.